.github/workflows/ci.yml

```diff
@@ -28,6 +27,8 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
         id: toolchain
       - run: rustup override set ${{steps.toolchain.outputs.name}}
+      - if: matrix.os == 'ubuntu-latest'
+        run: sudo apt-get -y install libfontconfig1-dev jq
```

Comment:
Not sure if jq is needed, but it's harmless. (It comes from here: https://github.com/yeslogic/fontconfig-rs/issues/42#issuecomment-2581931244)